components/gitpod-cli/cmd/git-token-validator.go

```diff
@@ -35,6 +36,13 @@ var gitTokenValidator = &cobra.Command{
 	Args:   cobra.ExactArgs(0),
 	Hidden: true,
 	Run: func(cmd *cobra.Command, args []string) {
+		log.SetOutput(io.Discard)
```

Comment:
@JanKoehnlein Does it make sense to you? Otherwise I am not sure whether everything get logged?

Reply:
Yes, it makes sense. I am just not sure whether we should use another file. The credential helper could still have the same file open for appending, as it starts the token validator in background and then releases the file on exit. Anyway, this shouldn't be a problem on Linux.